lib/asciidoctor-epub3/converter.rb

```diff
@@ -1765,6 +1765,11 @@ def class_string node
       def role_valid_class? role
         role.is_a? String
       end
+
+      def encode_alt_text node, val, to_attr = false
```

Comment:
I suggest naming this in a more generic way, like [encode_attribute_value](https://github.com/asciidoctor/asciidoctor/blob/v2.0.20/lib/asciidoctor/converter/html5.rb#L1318-L1320) from HTML5 converter. We also do not need to escape anything besides `"` because others are already escaped.

Reply:
Thanks for the suggestions! Will try to implement them and add unit tests. Mine was a quick WIP hack copying code from asciidoctor.

Reply:
Wait, we even already have a function for that: `xml_sanitize` does exactly what is needed.